Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
+
+gem 'activesupport', '~> 4.2.2'
+gem 'elasticsearch-model'
+gem 'activemodel'
+gem 'typhoeus'
+gem 'activerecord'
+
+group :test do
+  gem 'rspec', '~> 3.1.0'
+  gem 'rspec-activemodel-mocks', '~> 1.0.1'
+  gem 'elasticsearch-extensions'
+  gem 'rspec-expectations'
+  gem 'sqlite3'
+end
```

Comment:
I understood that gem dependencies should be declared primarily in the `.gemspec` file, with only loose version constraints (so as not to conflict with dependencies of the including apps). The Gemfile would then contain just a simple `gemspec` declaration referring to that file.

http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

These test dependencies would become "development" dependencies in that scheme.